Spring-2019/system_design_misc_coding/SOLUTIONS_2019Spring.md

```diff
@@ -36,21 +36,23 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. TinyURL
 
-Source: TODO :bug:
+Source: CTCI Chapter 9
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Design a service that produces and manages minified URLs.
```

Comment:
Might be nice to have a `common use case(s)` section, since it's probably one of the questions that will be asked.

Reply:
addressed in 7b4bca942b4bad6fca60f49e9660d468a8cd748a